src/libsystemd/sd-bus/bus-socket.c

```diff
@@ -948,7 +947,7 @@ int bus_socket_exec(sd_bus *b) {
         if (r < 0)
                 return -errno;
 
```

Comment:
there should be an assert on `b->busexec_pid == 0` at the beginning of bus_socket_exec(), so that we don't leak processes here if the bus address to connect to consists of multiple parts which we try one after the other.